pinot-core/src/main/java/org/apache/pinot/core/query/request/context/utils/QueryContextUtils.java

```diff
@@ -52,4 +58,79 @@ public static boolean isDistinctQuery(QueryContext query) {
     return aggregationFunctions != null && aggregationFunctions.length == 1
         && aggregationFunctions[0] instanceof DistinctAggregationFunction;
   }
+
+  /** Collect aggregation functions (except for the ones in filter). */
+  public static void generateTransforms(QueryContext queryContext, Set<String> postAggregations) {
```

Comment:
Rename it to `collectPostAggregations`? Same for other methods

Reply:
Done.